CIP-0165/format/format.ksy

```diff
@@ -0,0 +1,157 @@
+meta:
+  id: scls_file
+  title: Container for the cardano ledger state
+  file-extension: scls
+  ks-version: 0.9
+  endian: be
+
+doc: |
+
+  A seekable, versioned container for CBOR payloads whose structure is defined
+  by an external CDDL schema.
+
+seq:
+  - id: record
+    type: scls_record
+    repeat: eos
+
+types:
+  scls_record:
+    seq:
+      - id: size
+        type: u4
+        doc: size of the record
+      - id: record_type
+        type: u1
+        doc: type of the record
+      - id: record_data
+        doc: actual record data
+        type:
+          switch-on: record_type
+          cases:
+            0x00: rec_header
+            0x01: rec_manifest
+            0x10: rec_chunk
+  rec_header:
+    doc: Header block
+    seq:
+      - id: magic
+        contents: CDDL
+        doc: CDDL
+      - id: version
+        type: u2
+        doc: Version of the file format
+      - id: network_id
+        type: u1
+        enum: network_id
+        doc: Description of the network
+      - id: slot_no
+        type: u8
+        doc: Description of slot
+  rec_manifest:
+    doc: Manifest — is a trailer in the file that describes information of about the file contents
+    seq:
+      - id: total_entries
+        type: u8
+        doc: total amount of entries in the file
+      - id: total_chunks
+        type: u8
+        doc: total amount of chunks in the file
+      - id: summary
+        type: summary
+        doc: information about the file
+      - id: namespace_info
+        type: namespace_info
+        repeat: until
+        repeat-until: _.len_ns == 0
+        doc: information about the namespaces
+      - id: prev_manifest
+        type: u8
+        doc: absolute offset of the previous manifest, zero if there is no
+      - id: root_hash
+        type: digest
+        doc: merkle tree root of the live entries
+      - id: offset
+        type: u4
+        doc: relative offset to the beginning of the block
+  rec_chunk:
+    doc: Chunk - is a block with data
+    seq:
+      - id: seqno
+        type: u8
+        doc: Sequential number of the chunk
+      - id: format
+        type: u8
+        enum: chunk_format
+      - id: ns
+        type: tstr
+        doc: namespace
+      - id: len_data
+        type: u4
+        doc: size of payload
+      - id: data
+        size: len_data
+        doc: payload
```

Comment:
Could this be an entry of type `tstr`, or do we leave it like it is because of https://github.com/tweag/cardano-canonical-ledger/blob/2a6b2f7534912d2bee35b28c02bc4bab4bd19af5/scls-format/src/Cardano/SCLS/Internal/Record/Chunk.hs#L83 ?

Reply:
`tstr` requires encoding, and we have just binary data here, so it will not be correct. I assume